TMP2022/technical-writing-track/precious-tw-funke/README.md

```diff
@@ -0,0 +1,239 @@
+
+## Open An Account.
+
+![kuda-welcome-image](https://cdn.hashnode.com/res/hashnode/image/upload/v1663942031839/5DmnB8cCh.png)
+
+Creating an account on Kuda requires a valid email address, a phone number for verification, being over the age of 16, and an optional referral code. Do the following to create your Kuda account:
+
+1. Start the app on your mobile device.
+2. Select "Open An Account."
+3. Enter a valid email address.
+4. Enter a password of your choosing.
+5. Verify your password.
+6. Enter the referral code (optional).
+7. An email with an email verification code is sent to the email address you specified.
+8. Enter the code you received and tap OK.
+
+You will be redirected to a new page where you must enter your personal information into the input fields as follows:
+
+- First name
+- Middle name
+- Last name
+- Gender
+- Birthdate
+- Telephone number
+- Submit Verification code received on your telephone number
+- Finally, you must enter your current address
+
+Following this is a "Welcome to Kuda" pop-up. But it doesn't stop there. You will be directed to the login screen, where you must enter your email and password.
+
+Now you're logged into your kuda account. The next step in setting up your kuda account is setting a passcode. A **Passcode** is a 6-digit code only used to sign in to the kuda app on your mobile device.
+
+When all of this is done you'd have to wait a bit for your account to be created (your account status shows pending till your account has been created).
+
+## Verify Identity
+
+Kuda asks you to perform two actions to verify your identity
+
+- Input your valid Bank Verification Number (BVN).
+- Record a short video.
+
+Kuda requests for your BVN so they can be sure no one is pretending to be you.
+When you confirm your BVN, Kuda matches your details with the information on the national BVN database owned by the Central Bank of Nigeria. Other of your account(s) will not be accessed with your BVN.
+Secondly, Kuda uses Artificial intelligence to record your face.
+
+## Secure Your Account
+
+Kuda will request that you create a transaction pin. Your transaction pin is a 4-digit pin that allows you perform monetary transactions across kuda.
+
+_N/B: your transaction pin and your passcode are not the same thing. The former is for performing transactions while the latter is for accessing your kuda dashboard._
+
+## More About You
+
+This section focuses on how you intend to use kuda. You could, for example, use kuda for everyday transactions such as savings, debit card, and so on. It also asks questions such as how much you earn and what type of work you do in order to provide you with a wholesome experience when using Kuda Bank.
+
+When the more about survey is completed, you can access your dashboard and start transacting.
+
+# EXISTING USERS
```

Comment:
Check to use the proper heading here. Is this supposed to be a heading One or Two?

Reply:
Restructured the file and made it a h3.